app/views/student/form.php

```diff
@@ -35,4 +614,66 @@
       <input type="submit" name="edit" value="Update Profile" />
     </form>
   </div>
-</panel>
\ No newline at end of file
+</panel>
+
+<script type="text/javascript">
+function educationValueChanged() {
+  if($('.undergraduate').is(":checked"))
+  {
+    $("#uDegreeChooser").prop('required', true);
+    $("#uYearChooser").prop('required', true);
+    $("#uGraduationMonthChooser").prop('required', true);
+    $("#uGraduationYearChooser").prop('required', true);
+    $("#gDegreeChooser").prop('required', false);
+    $("#gYearChooser").prop('required', false);
+    $("#gGraduationMonthChooser").prop('required', false);
+    $("#gGraduationYearChooser").prop('required', false);
+    $(".undergraduateShow").show();
+    $(".graduateShow").hide();
+  }
+  else
+  {
+    $("#uDegreeChooser").prop('required', false);
+    $("#uYearChooser").prop('required', false);
+    $("#uGraduationMonthChooser").prop('required', false);
+    $("#uGraduationYearChooser").prop('required', false);
+    $("#gDegreeChooser").prop('required', true);
+    $("#gYearChooser").prop('required', true);
+    $("#gGraduationMonthChooser").prop('required', true);
+    $("#gGraduationYearChooser").prop('required', true);
+    $(".graduateShow").show();
+    $(".undergraduateShow").hide();
+  }
+}
+
+function lookingForValueChanged() {
+    if($('.internship').is(":checked") || $('.fulltime').is(":checked"))
```

Comment:
The code below seems like it can be simplified a lot. Rather than doing each other individually, you can define a class that when checked, shows/hides another class with a certain tag to indicate which one it is showing/hiding.